requirements.txt

```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
 boto3>=1.35.96
-click>=8.1.7,<9.0.0
+click>=8.1.7
```

Comment:
8.1.8 seems to be the last version that works

Reply:
I misunderstood this diff. You're just keeping the minimal version at 8.1.7, not going to 8.1.8, but removing the upper bound. Got it.